src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/VariationType.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL.Types;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.InventoryModule.Core.Model.Search;
+using VirtoCommerce.InventoryModule.Core.Services;
+using VirtoCommerce.XDigitalCatalog.Specifications;
+
+namespace VirtoCommerce.XDigitalCatalog.Schemas
+{
+    public class VariationType : ObjectGraphType<ExpVariation>
+    {
+        public VariationType(IProductInventorySearchService productInventorySearchService)
+        {
+            Field<StringGraphType>(
+                "id",
+                description: "Id of variation.",
+                resolve: context => context.Source.Product.Id
+            );
+
+            Field<StringGraphType>(
+                "code",
+                description: "SKU of variation.",
+                resolve: context => context.Source.Product.Code
+            );
+
+            // TODO: change to connection
+            FieldAsync<AvailabilityDataType>("availabilityData", resolve: async context =>
+            {
+                var product = context.Source.Product;
+
+                var invntorySearch = await productInventorySearchService.SearchProductInventoriesAsync(new ProductInventorySearchCriteria
```

Comment:
`DataLoaders ` is missed you will have performance issues called `N+1 problem`  and extra round-trips requests for when multiple products are requested